pkg/secrets/check_rights_windows.go

```diff
@@ -117,3 +110,150 @@ func checkRights(filename string, allowGroupExec bool) error {
 	}
 	return nil
 }
+
+func getACL(filename string) (*winutil.Acl, error) {
```

Comment:
Mine adding some comments to explain what's going on here - most of the team is linux focused, and I had to spend a bit of time trying to understand what's going on.

Reply:
Yes, for sure.